azure-iot-device/tests/conftest.py

```diff
@@ -26,7 +26,7 @@
 """
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture  # (scope="function")
```

Comment:
I assume this is a leftover.  function scope is default, so this isn't really necessary anyway (also on line 38 if you're going to remove this)